@shared/api/types.ts

```diff
@@ -240,13 +240,6 @@ export interface Balance {
 }
 
 export type BlockAidScanAssetResult = Blockaid.TokenScanResponse;
-export interface BlockAidScanAssetResultIndex {
```

Comment:
@piyalbasu I removed this because Blockaid removed `Blockaid.TokenScanResponse.BasicMetadataToken` in v0.26 and this looks unused. Any reason to keep this type around?